config/setupJest.js

```diff
@@ -8,6 +8,8 @@ import sinonStubPromise from 'sinon-stub-promise';
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import i18next from 'i18next';
+// TODO remove next line after upgrading node version to at least 7
+import 'babel-polyfill';
```

Comment:
Jenkins failed with `JavaScript heap out of memory`

https://jenkins.lisk.io/job/lisk-hub/job/PR-1475/1/consoleFull

Reply:
change done